src/current/v25.2/create-policy.md

```diff
@@ -0,0 +1,115 @@
+|-------------------------------------|-----------------------------------------------------|--------------------------------------------------|---------------------------------------------------|-------------------------------------------------------|-----------------------------------------------|
+| `SELECT`                            | ✓                                                   | —                                                | —                                                 | —                                                     | —                                             |
+| `SELECT ... FOR UPDATE / FOR SHARE` | ✓                                                   | —                                                | ✓                                                 | —                                                     | —                                             |
+| `INSERT`                            | —                                                   | ✓                                                | —                                                 | —                                                     | —                                             |
+| `INSERT ... RETURNING`              | ✓ ‡                                                 | ✓                                                | —                                                 | —                                                     | —                                             |
+| `UPDATE`                            | ✓ ‡                                                 | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| `DELETE`                            | ✓ ‡                                                 | —                                                | —                                                 | —                                                     | ✓                                             |
+| `INSERT ... ON CONFLICT DO UPDATE`  | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| `UPSERT`                            | ✓                                                   | —                                                | ✓                                                 | ✓                                                     | —                                             |
+
+‡ The `SELECT` check is only evaluated when the statement actually needs to read from the relation, e.g., in a `WHERE`, `SET`, or `RETURNING` clause that references table columns.
```

Comment:
We should probably leave this part out until we fix this (i.e. 145894). What's process to ensure this is added back and the additional consideration below is updated when that change goes it? Do we just need to ensure the commit has a release note filled in?

Reply:
>  We should probably leave this part out until we fix this (i.e. cockroachdb/cockroach#145894)

makes sense, removed this note for now 

> What's process to ensure this is added back and the additional consideration below is updated when that change goes it? Do we just need to ensure the commit has a release note filled in?

yes, if the release note is filled in, an issue will be created in the DOC queue whenever the change gets done